ibis/expr/api.py

```diff
@@ -10,6 +10,7 @@
 import dateutil.parser
 import pandas as pd
 import toolz
+from pandas.core.tools.datetimes import to_datetime, to_time
```

Comment:
use `pd.to_datetime`, unfortunatly `to_time` is not exposed directly

Reply:
I am not sure if it was you were suggesting, but I've  changed `to_time(value)` with `pd.to_datetime(value).time()`
Is it what you were referring to?

Reply:
hmm that's not what i meant, but if it works would be happier with this (as `to_time`) is not exposed in the `pd` namespace until 1.2 IIRC but i guess it worked.